slack_sdk/web/client.py

```diff
@@ -4889,6 +4889,249 @@ def search_messages(
+        """Deletes an item from an existing List.
+        https://docs.slack.dev/reference/methods/slacklists.items.delete
+        """
+        kwargs.update(
+            {
+                "list_id": list_id,
+                "id": id,
+            }
+        )
+        kwargs = _remove_none_values(kwargs)
+        return self.api_call("slackLists.items.delete", json=kwargs)
+
+    def slackLists_items_deleteMultiple(
+        self,
+        *,
+        list_id: str,
+        ids: List[str],
+        **kwargs,
+    ) -> SlackResponse:
+        """Deletes multiple items from an existing List.
+        https://docs.slack.dev/reference/methods/slacklists.items.deletemultiple
+        """
+        kwargs.update(
+            {
+                "list_id": list_id,
+                "ids": ids,
+            }
+        )
+        kwargs = _remove_none_values(kwargs)
+        return self.api_call("slackLists.items.deleteMultiple", json=kwargs)
+
+    def slackLists_items_info(
+        self,
+        *,
+        list_id: str,
+        id: str,
+        include_is_subscribed: Optional[bool] = None,
+        **kwargs,
+    ) -> SlackResponse:
+        """Get a row from a List.
+        https://docs.slack.dev/reference/methods/slacklists.items.info
+        """
+        kwargs.update(
+            {
+                "list_id": list_id,
+                "id": id,
+                "include_is_subscribed": include_is_subscribed,
+            }
+        )
+        kwargs = _remove_none_values(kwargs)
+        return self.api_call("slackLists.items.info", json=kwargs)
+
+    def slackLists_items_list(
+        self,
+        *,
+        list_id: str,
+        limit: Optional[int] = None,
+        cursor: Optional[str] = None,
+        archived: Optional[bool] = None,
+        **kwargs,
+    ) -> SlackResponse:
+        """Get records from a List.
+        https://docs.slack.dev/reference/methods/slacklists.items.list
+        """
+        kwargs.update(
+            {
+                "list_id": list_id,
+                "limit": limit,
+                "cursor": cursor,
+                "archived": archived,
+            }
+        )
+        kwargs = _remove_none_values(kwargs)
+        return self.api_call("slackLists.items.list", json=kwargs)
+
+    def slackLists_items_update(
+        self,
+        *,
+        list_id: str,
+        cells: List[Dict[str, Any]],
+        **kwargs,
+    ) -> SlackResponse:
+        """Updates cells in a List.
+        https://docs.slack.dev/reference/methods/slacklists.items.update
+        """
+        kwargs.update(
+            {
+                "list_id": list_id,
+                "cells": cells,
+            }
+        )
+        kwargs = _remove_none_values(kwargs)
+        return self.api_call("slackLists.items.update", json=kwargs)
+
+    def slackLists_update(
+        self,
+        *,
+        id: str,
+        name: Optional[str] = None,
+        description_blocks: Optional[List[Dict[str, Any]]] = None,
```

Comment:
🧱 suggestion: Since this is also rich text we can perhaps add that block as an option too?